src/libraries/System.Text.Json/gen/TypeGenerationSpec.cs

```diff
@@ -59,8 +59,8 @@ internal class TypeGenerationSpec
         public TypeGenerationSpec? NullableUnderlyingTypeMetadata { get; private set; }
 
         /// <summary>
-        /// Supports deserialization of extension data dictionaries typed as I[ReadOnly]Dictionary<string, object/JsonElement>.
-        /// Specifies a concrete type to instanciate, which would be Dictionary<string, object/JsonElement>.
+        /// Supports deserialization of extension data dictionaries typed as I[ReadOnly]Dictionary&lt;string, object/JsonElement&gt;.
+        /// Specifies a concrete type to instanciate, which would be Dictionary&lt;string, object/JsonElement&gt;.
```

Comment:
@eiriktsarpalis and I think this one can also be wrapped:

```suggestion
        /// Supports deserialization of extension data dictionaries typed as <c>I[ReadOnly]Dictionary&lt;string, object/JsonElement&gt;</c>.
        /// Specifies a concrete type to instanciate, which would be <c>Dictionary&lt;string, object/JsonElement&gt;</c>.
```